commands/utils/testsutils.go

```diff
@@ -39,7 +39,18 @@ func PrepareTestEnvironment(t *testing.T, projectName, testDir string) (string,
 	err = fileutils.CopyDir(filepath.Join("testdata", testDir), tmpDir, true, []string{})
 	assert.NoError(t, err)
 
-	restoreDir, err := Chdir(filepath.Join(tmpDir, projectName))
+	// Copy Git folder as .git if exists
+	currentDir := filepath.Join(tmpDir, projectName)
+	gitPath := filepath.Join(currentDir, "git")
```

Comment:
Should it be ".git"?
```suggestion
	gitPath := filepath.Join(currentDir, ".git")
```

Reply:
As test data we save this as `git` and when we copy we copy that as `.git`